wishlists.py

```diff
@@ -170,14 +170,29 @@ def clear_wishlist(wishlist_id):
                 return jsonify(message), HTTP_404_NOT_FOUND
         return db.retrieve_wishlist(wishlist_id), HTTP_200_OK
     except WishlistException:
+
         message = { 'error' : 'Wishlist %s was not found' % wishlist_id }
         return jsonify(message), HTTP_404_NOT_FOUND
 
 
+@app.route('/wishlists/search', methods=['GET'])
+def search_wishlists():
+    """
+    The route for searching items with specific keyword or ID.
+    """
+    try:
+        query = request.args.get('q', None)
+        uid = request.args.get('user_id',None)
+        print query
+        print uid
```

Comment:
Please remove any print statements in the server code - we will want to log stuff later on for sure, but it's best to write to a file than to standard out.